src/components/map/layers/NetworkLayers.tsx

```diff
@@ -155,7 +155,8 @@ export default class NetworkLayers extends Component<INetworkLayersProps> {
         if (properties.soltyyppi !== NodeType.STOP) return;
 
         const routePathLinks =
-            await RoutePathLinkService.fetchLinksWithLinkStartNodeId(properties.soltunnus);
+            await RoutePathLinkService.fetchAndCreateRoutePathLinksWithStartNodeId(
```

Comment:
Same function, but tried to improve name so it would describe more what it actually does